src/styles/cylc/_toolbar.scss

```diff
@@ -26,20 +26,39 @@
 .c-toolbar {
   background-color: map-get($grey, 'lighten-4') !important;
 
+  .v-toolbar__title {
+    // clear default padding (too much space on the left)
+    padding: 0!important;
```

Comment:
This gets rid of the extra large space between the collapsed "drawer" icon and the flow name.

Reply:
Do we need a space between `0` and `!important`?

Reply:
Question answered, looks like webpack or the browser add the space
![image](https://user-images.githubusercontent.com/304786/114943729-8f399580-9e9a-11eb-9a6a-232f5e6c26df.png)